src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int ARRAY_EXTEND_COEFFICIENT = 2;
+
+    private int arrayCapacity = DEFAULT_CAPACITY;
+    private int threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
```

Comment:
same here, you rely on its value only one time in `put` method, the other two mentions are setting it :) this only clutters the code